docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -43,13 +43,21 @@ component "prometheus-operator" {
     }
   }
 
-  prometheus_metrics_retention = "14d"
-  prometheus_external_url      = "https://api.example.com/prometheus"
-  prometheus_node_selector = {
-    "kubernetes.io/hostname" = "worker3"
+  prometheus {
```

Comment:
This commit should be splitted and should go to appropriate commits:
- move parameters to block
- add ingress block